BraveShared/Analytics/DAU.swift

```diff
@@ -0,0 +1,237 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import XCGLogger
+
+private let log = Logger.browserLogger
+
+public class DAU {
+
+    /// Default installation date for legacy woi version.
+    public static let defaultWoiDate = "2016-01-04"
+    
+    private static let apiVersion = 1
+    private static let baseUrl = "https://laptop-updates.brave.com/\(apiVersion)/usage/ios?platform=ios"
```

Comment:
I know this was a direct port but hardcoded URLs make me afraid lol

Reply:
Was thinking the same thing, you think appending specific path components better?

Reply:
In the past we put it into like build configs or plist's so in the off-case QA needed a build pointing to different URLs we didn't have to make any changes to the code.

In this case I'm not sure if its really needed or not, but worth a thought down the road

Reply:
We use the same url for production.debug atm, and we pass different param depending on release channel

Reply:
Yes, but agree, we can probably abstract, worth a convo, but I'm personally fine with this for rn, if y'all are.